gxapi_rails.gemspec

```diff
@@ -25,16 +25,17 @@ Gem::Specification.new do |s|
   s.add_dependency(%q<rest-client>, [">= 0"])
   s.add_dependency(%q<json>, [">= 0"])
   s.add_dependency(%q<google-api-client>, [">= 0"])
+  s.add_dependency(%q<test-unit>, [">= 0"])
```

Comment:
What is depending on it?  Seems weird to have a test framework that we aren't actually using as a runtime dependency.